dot/state/epoch.go

```diff
@@ -272,8 +353,29 @@ func (s *EpochState) setLatestConfigData(epoch uint64) error {
 	return s.db.Put(latestConfigDataKey, buf)
 }
 
-// GetConfigData returns the BABE config data for a given epoch
-func (s *EpochState) GetConfigData(epoch uint64) (*types.ConfigData, error) {
+// GetConfigData returns the config data for a given epoch persisted in database
+// otherwise will try to get the data from the in-memory map using the header
+// if the header params is nil then it will search only in database
```

Comment:
nit

```suggestion
// GetConfigData returns the config data for a given epoch persisted in the database
// and otherwise tries to get the data from the in-memory map using the header.
// If the header params is nil then it will search only in the database
```